Components/Utils/fetchArticles.js

```diff
@@ -1,10 +1,17 @@
 import axios from "axios";
 
-function fetchArticles(){
+function fetchArticles(topic){
+  if(topic){
     return axios
-      .get("https://be-nc-news-d5oe.onrender.com/api/articles").then(({data}) => {
+      .get(`https://be-nc-news-d5oe.onrender.com/api/articles?topic=${topic}`).then(({data}) => {
         return data.articles
       })

```

Comment:
You don't have to do this conditional logic checking if `topic` exists first when using Axios, check out [these notes](l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=optional-queries) on how to make a request using optional queries